blockchains/erc20/erc20_worker.js

```diff
@@ -103,8 +105,10 @@ class ERC20Worker extends BaseWorker {
     let overwritten_events = [];
     const timestampsCache = new TimestampsCache();
     if ('extract_exact_overwrite' === constants.CONTRACT_MODE) {
-      events = await getPastEvents(this.web3, result.fromBlock, result.toBlock, this.allOldContracts, timestampsCache);
-      changeContractAddresses(events, this.contractsOverwriteArray);
+      if (this.allOldContracts.length > 0) {
```

Comment:
This array would be empty in some of the modes. Providing empty array would return all events.